spec/project_printer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "spec_helper"
+require "project_printer"
+
+RSpec.describe ProjectPrinter do
+  describe "#changed_name" do
+    it "changes the project name" do
```

Comment:
I think the spec description is misleading. I wouldn't expect the printer to do the actual name change logic.

In fact, I think most of the names/descriptions in this spec should be reviewed.

Reply:
Good point. Done!